index.html

```diff
@@ -2,10 +2,12 @@
 <html>
```

Comment:
Should we work toward making the template such that we can add a comment here along the lines of the following?

```
<!-- In general, you should not need to edit this file. When in doubt, ask! -->
```